docker-entrypoint.sh

```diff
@@ -54,7 +60,7 @@ else
   $PGCTL -D "$PGDATA" -m fast -w stop
 fi
 
-touch "$PGDATA/initialized"
+touch "${PGDATA}initialized"
```

Comment:
Isn't better to have it as before for readability?

Reply:
it is, but string manipulation of PGDATA was easier this way
We may improve it if required